src/compiler/crystal/semantic/flags.cr

```diff
@@ -11,6 +12,10 @@ class Crystal::Program
     @flags ||= flags_for_target(codegen_target)
   end
 
+  def host_flags
+    @host_flags ||= flags_for_target(Config.default_target)
```

Comment:
I'd also suggest to rename `Config.default_target` to `Config.host_target` because that's what it is.

Reply:
OK, didn't turn out *too* big.